generator/app.go

```diff
@@ -55,13 +60,93 @@ const (
 
 	regexRawGitHubURL = "https://raw.githubusercontent.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/(?P<branch>[^/]+)/(?P<path>.*)"
 	regexGitHubURL    = "https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/(blob|tree)/(?P<branch>[^/]+)/(?P<path>.*)"
+
+	// For KEPs automation
+	alpha  = "alpha"
+	beta   = "beta"
+	stable = "stable"
+	kepURL = "https://storage.googleapis.com/k8s-keps/keps.json"
 )
 
 var (
 	baseGeneratorDir = ""
 	templateDir      = "generator"
+	releases         = Releases{}
 )
 
+// KEP represents an individual KEP holding its metadata information.
+type KEP struct {
+	Name            string `json:"name"`
+	Title           string `json:"title"`
+	KepNumber       string `json:"kepNumber"`
+	OwningSig       string `json:"owningSig"`
+	Stage           string `json:"stage"`
+	LatestMilestone string `json:"latestMilestone"`
+}
```

Comment:
Similar to the above, the KEP structure is also present in https://github.com/kubernetes/enhancements/blob/9d63597d2d85a0ffba5bb4cee0172157948168ce/api/proposal.go#L94

Reply:
+1. Resolved in the refreshed commit.